documentation/milestones/individual_milestones/static_site_improvements.md

```diff
@@ -0,0 +1,208 @@
+
+### Destination for live updating metrics
+<!-- Required -->
+
+## Planning
+
+### Assumptions & dependencies
+<!-- Required -->
+
+What functionality do we expect to be in place ***before*** work starts on this deliverable?
+
+- **Front-end:** These improvements will build on the front-end work completed in the [initial static site launch](https://github.com/HHS/grants-equity/issues/62) which delivered the following functionality:
+  - **[Front-end CI/CD](https://github.com/HHS/grants-equity/issues/58):** Automatically tests and deploys front-end code
+  - **[Foundational UI](https://github.com/HHS/grants-equity/issues/60):** Enforces a consistent user interface and web design system across the frontend
+
+What functionality do we expect to be in place by ***the end*** of work on this deliverable?
+
+- **[Content Management](https://github.com/HHS/grants-equity/issues/75):** Enables non-technical users to either directly make content updates to the site or to propose and review content changes that are implemented by the engineering team.
+- **[Internationalization](https://github.com/HHS/grants-equity/issues/64):** Enables users to change the language in which they view site contents
+- **[Translation Process](https://github.com/HHS/grants-equity/issues/81):** Facilitates and publishes translations of front-end content in multiple languages
+- **[Feedback Mechanism](https://github.com/HHS/grants-equity/issues/596):** Enables users to provide feedback or ask questions directly from the site
+
+Is there any notable functionality we do ***not*** expect to be in place before works starts on this deliverable?
+
+- **API:** Work on the API will be completed as part of [a separate deliverable](https://github.com/HHS/grants-equity/issues/70), so functionality that requires the API is explicitly out of scope in this deliverable.
+
+### Open questions
+<!-- Optional -->
+
+*What existing services does the federal government provide for translation?*
+
+TODO
+
+### Not doing
+<!-- Optional -->
+
+The following work will *not* be completed as part of this milestone:
+
+1. **Search Functionality:** Because search requires the API, site improvements in this deliverable will ***not*** include any search functionality. That will be completed in a [search-specific deliverable](https://github.com/HHS/grants-equity/issues/89)
+
+## Integrations
+
+### Translations
+<!-- Required -->
+
+*Does this milestone involve delivering any content that needs translation?*
+
+TODO
+
+*If so, when will English-language content be locked? Then when will translation be started and completed?*
+
+TODO
+
+### Services going into PROD for the first time
+<!-- Required -->
+
+This can include services going into PROD behind a feature flag that is not turned on.
+
+1. TODO
+
+### Services being integrated in PROD for the first time
+<!-- Required -->
+
+Are there multiple services that are being connected for the first time in PROD?
+
+1. TODO
```

Comment:
```suggestion
There may be, depending on how we decide to manage content, translate, or collect feedback from users. 
```

Reply:
Once our team breaks down the work and thinks about approach, we could add it in this deliverables doc. Some of the work in this deliverable may answer a lot of the questions - what data is exposted fo the first time, etc.